src/helpers/check-merge-safety.ts

```diff
@@ -21,6 +21,7 @@ import { map } from 'bluebird';
 import { setCommitStatus } from './set-commit-status';
 import * as core from '@actions/core';
 
+const maxBranchNameLength = 50;
```

Comment:
Just curious, did the linter get mad at defining this after usage?

Reply:
nah didn't see anything come up